spring-cloud/tracing/customer-service/src/main/java/com/example/demo/Controller.java

```diff
@@ -27,9 +27,7 @@ public ResponseEntity<Customer> getCustomer(@PathVariable("id") long customerId)
     customer.setId(customerId);
     customer.setName(NAMES.get((int) customerId));
 
-    throw new IllegalStateException("BWAAAH!");
```

Comment:
This exception was being thrown for all calls to `http://localhost:8080/customers/<ID>`.

Reply:
oops, forgot to take that out (I used it for testing).